sandworm/bin/main.ml

```diff
@@ -6,11 +6,15 @@ let main commit has_certificate =
   let daily_bundle =
     Metadata.(Bundle.create_daily ~commit ~has_certificate Target.defaults)
   in
-  let daily_bundle_date = Metadata.Bundle.get_data_string_from daily_bundle in
+  let daily_bundle_date = Metadata.Bundle.get_date_string_from daily_bundle in
   let s3_daily_bundle = Filename.concat Config.s3_bucket_ref daily_bundle_date in
   let () =
     Rclone.copy ~config_path:Config.rclone_path Config.artifacts_path s3_daily_bundle
   in
+  let latest_bundle = Filename.concat Config.s3_bucket_ref "latest" in
+  let () =
+    Rclone.copy ~config_path:Config.rclone_path Config.artifacts_path latest_bundle
+  in
```

Comment:
I don't know if that is possible on a S3 bucket content leve but I think it would be preferable to me if we did a `30x` redirect on the `/latest` to the latest date instead of changing the content where latest is pointing to.

Reply:
Its not a DNS thing, it's a HTTP thing. It can be [configured on S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/how-to-page-redirect.html) but not from the bucket contents itself.

Reply:
Yes, but here we are not using the S3 interface anymore. The first iteration is of the work was using _RClone_ with an S3 bucket. This version is using _RClone_ with SFTP. I kept the S3 wording, as the _RClone_ tends to refer a lot to it in the documentation. 

In terms of architecture, the website is served using GitHub Pages whereas the binaries and certificates are served using Caddy (https://caddyserver.com/docs/quick-starts/static-files). As a result, we don't have something related to S3. The only access we have from our perspective is the file system structure.

TL;DR: unfortunately, we can't do this in the current architecture :smile: